Automattic/woocommerce-payments
# Fix GPay button missing in Safari

Fixes #9202

#### Changes proposed in this Pull Request

This update fixes the missing Google Pay (GPay) button in Safari in the blocks checkout and cart pages. 
The issue occurred because the shortcode checkout had Apple Pay and Google Pay always enabled, while the Blocks version left Stripe to decide, causing the GPay button to not appear.

#### Testing instructions

To test the changes, follow these steps on a publicly accessible URL and an HTTPS-enabled website:

1. On the `develop` branch, verify that only the Apple Pay button is visible by visiting the Blocks checkout page with a product in the cart using Safari.
2. Switch to this branch, repeat the test, and confirm that both the Apple Pay and GPay buttons are now displayed in Safari.
3. In the admin area, go to the Express Checkout preview in the WooPayments settings and confirm that both buttons are shown in Safari.

-------------------


**Post merge**
